comp/otelcol/status/impl/status_test.go

```diff
@@ -0,0 +1,66 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+package statusimpl
+
+import (
+	"bytes"
+	"go.uber.org/fx"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/DataDog/datadog-agent/comp/core/config"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+)
+
+type testDependencies struct {
+	fx.In
+	Config config.Component
+}
+
+func TestStatusOut(t *testing.T) {
+	testDeps := fxutil.Test[testDependencies](t, config.MockModule())
+	deps := Dependencies{
+		Config: testDeps.Config,
+	}
+	provides := NewComponent(deps)
```

Comment:
Please use the the direct constructor for mocks (cf https://datadoghq.dev/datadog-agent/components/testing/).

Here simply use [this](https://github.com/DataDog/datadog-agent/blob/main/comp/core/config/config_mock.go#L58):
```suggestion
	provides := NewComponent(Dependencies{
			Config: config.NewMock(t),
	})
```